docs/adr/adr-007-incentivized-testnet.md

```diff
@@ -0,0 +1,100 @@
+# ADR #007: Incentivized Technical Requirements ADR
+
+## Changelog
+
+* 2022-04-15: proposed
+
+## Authors
+
+@renaynay @Wondertan
+
+## Context
+This ADR serves as a tracking document for celestia-node's technical requirements for the planned incentivized testnet.
+
+All epics/tracking issues have been linked in the sub-headers of [technical requirements](#technical-requirements).
+
+Top-level features will include a link to the epic-style tracking issue as well which can be used for a further detailed
```

Comment:
nit, but can we avoid hard line breaks in markdown files? I use side-by-side diffs and it looks atrocious at the particular width chosen (90??).